cluster/gce/config-default.sh

```diff
@@ -24,8 +24,8 @@ ZONE=${KUBE_GCE_ZONE:-us-central1-b}
 REGION=${ZONE%-*}
 RELEASE_REGION_FALLBACK=${RELEASE_REGION_FALLBACK:-false}
 REGIONAL_KUBE_ADDONS=${REGIONAL_KUBE_ADDONS:-true}
-NODE_SIZE=${NODE_SIZE:-n1-standard-2}
-NUM_NODES=${NUM_NODES:-3}
+NODE_SIZE=${NODE_SIZE:-n1-standard-32}
+NUM_NODES=${NUM_NODES:-16}
```

Comment:
Will do, this was part of the kubemark tests you had suggested and will not be pushed.